swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, id=None, condition= 0):
+        # self.id = id if id is not None else uuid.uuid4().int
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+
+    def get_category(self):
+        return "Item"
```

Comment:
While this works, it means you are going to have to rewrite your `get_category() method for each of your child classes. The idea behind inheritance is to try and figure out which of your parent functions could be passed down to its children successfully! In this case, we have a way of retrieving the class specific name that would allow us to pass this method down without having to hard code the category for each class!

```suggestion
    def get_category(self):
        return self.__cls__.__name__
```
If one of the child classes calls this inherited method, it will return the appropriate class name!